File: app/src/main/java/jgappsandgames/smartreminderslite/home/HomeAdapter.java
package jgappsandgames.smartreminderslite.home;

// App
import jgappsandgames.smartreminderslite.adapter.TaskAdapterInterface;

// Save
import jgappsandgames.smartreminderssave.tasks.TaskManager;

/**
 * HomeAdapter
 * Created by joshua on 8/31/17.
 */
class HomeAdapter extends TaskAdapterInterface {
    // Initializer
    HomeAdapter(HomeActivity activity) {
        super(activity, activity, TaskManager.home, null);
    }
}

File: app/src/main/java/jgappsandgames/smartreminderslite/home/HomeAdapter.kt
package jgappsandgames.smartreminderslite.home

// App
import jgappsandgames.smartreminderslite.adapter.TaskAdapterInterface

// Save
import jgappsandgames.smartreminderssave.tasks.TaskManager

/**
 * HomeAdapter
 * Created by joshua on 12/13/2017.
 *
 * Adapter for the Task List in the Home View
 */
class HomeAdapter(activity: HomeActivity): TaskAdapterInterface(activity, activity, TaskManager.home, null)